packages/table/src/table.tsx

```diff
@@ -542,6 +542,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             this.bodyElement,
             this.grid,
         );
+        this.locator = locator;
 
         const viewportRect = locator.getViewportRect();
         this.setState({ locator, viewportRect });
```

Comment:
@gscshoyru that doesn't seem like a reason to keep it in state, vs a single instance member created at startup time. no reason to block _this_ PR, but we should follow up on this.